telco-hub/configuration/reference-crs/optional/logging/cLusterLogSubscription.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
The filename of this CR is still mis-capitalized... telco-hub/configuration/reference-crs/optional/logging/cLusterLogSubscription.yaml

Reply:
i see why it was not updated. i changed the fine name since no content was not updated it did not take it. locally it was changed. now it should be ok, sorry